README.md

```diff
@@ -1090,7 +1092,29 @@ There are two css classes (`event` and `message`) that should be applied to mess
 
 #### Miro
 
-Another option is using [Miro](https://miro.com/). Miro allows the creation of diagrams in a whiteboard that can be exported and imported as a backup file (RTB) by a board owner. Diagrams created in Miro can be used to generate images that should be committed along with whiteboard backup. The idea is that the whiteboard with the diagrams can be imported in Miro by any team and evolved along with the documentation. To generate diagram images from Miro board, surround the diagram with a frame, select the frame, select the "Export as image" option and export the image as PNG.
+Another option is [Miro](https://miro.com/). Miro allows the creation of diagrams on a whiteboard. A board can be saved as a board backup file (.rtb) and can be used to create images.
+
+When creating diagram images consider **not** using transparent backgrounds. When using a transparent background, if users have a dark theme enabled, the lines in the diagram can become hard to see creating a bad user experience while reading our documentation.
+
+To create an image:
+
+- Drag a rectangle around the relevant objects on the board
+- From the “three dots” (…) menu above the selection:
+  - Click “Create frame”
+  - Click “Export as image”
+- Choose the appropriate JPG size and click "Export"
+
+If you want to save it as a `.png`, copy the image to an image editing tool like "paint" and save it as a `.png`.
+
+Alternatively, if you choose to create a draw.io diagram, Miro will allow you to export to `.png`:
```

Comment:
Again, rather than "if you choose", it feels like we should give some guidance as to whether to use draw.io within Miro or not. What are the reasons for doing so?

Reply:
I'd rather use draw.io as it allows the diagrams to be cleanly built and it was easier to take advantage of the shapes libraries for architecture than using Miro's libraries which are locked with our current subscription level. However, on my TF there were split opinions as some people felt it was enough to just import icons from the internet as needed. Personally, I'd rather use draw.io within Miro due to the libraries it has available and how easy it is to convert images and to not allow people to accidentally change diagrams without realizing it.